.circleci/config.yml

```
@@ -28,12 +28,12 @@ jobs:
 
   deploy-to-prod:
     docker:
-      - image: circleci/node:7.10
+      - image: circleci/node:12.6
     working_directory: ~/repo
 
     steps:
       - checkout
-      - run: echo "This would deploy to prod"
+      - run: echo "This would deploy to prod.."
```

Can fix later. Typo.
```suggestion
      - run: echo "This would deploy to prod."
```